src/components/CareerFramework/Group.js

```diff
@@ -0,0 +1,23 @@
+import React, { Fragment } from 'react'
+import generate from 'shortid'
+
+import Hr from '../Common/Hr'
+
+import Level from './Level'
+
+const Group = props => {
+  const { levels } = props
```

Comment:
Can make it one line as `const Group = ({ levels }) => {`

Reply:
and even the need for the keyword `return`
`const Group = ({ levels }) => levels && levels.length && levels.map(( \\ rest of the code`